### ISSUE TITLE
Container app autoscale with Concurrent requests - Functionality clarification

### ISSUE THREAD

**Issue by NickRadev** [microsoft/azure-container-apps#554]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is a: (mark with an x)
- [ ] bug report -> please search issues before submitting
- [x] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)

### Issue description
A clear and concise description of the observed issue.
- Whenever a container app is set to autoscale based on HTTP concurrent requests rather than scaling as needed it scales to the max set replica count. Example:
- Min is 1 and Max is 10
- Concurrent request trigger set to 100 concurrent requests
- 200 requests come in
- The behavior that's seen is that instead of autoscaling to 2 replicas it scales to the max of 10
### Steps to reproduce
The issue reproduces every time the HTTP autoscale is configured as explained above

**Expected behavior** Autoscaling to accommodate the new volume as needed

**Actual behavior** Autoscaling to the max available replicas when there's no need

**Screenshots**         
N/A

### Additional context
Submitting this issue was suggested by Mike Morton (Microsoft) internally.

**Comment by BigMorty** [comment]
Thanks for reporting this. We have a PR out to fix this issue.

**Comment by NickRadev** [comment]
Hi @BigMorty, thanks for the fast update. Just to confirm, if I'm understanding correctly this isn't intended behavior as it should be behaving like the other autoscale rules where they scale replicas based on usage is that right? Additionally is there an ETA on the fix?

**Comment by ahmelsayed** [comment]
This should be fixed. In the examples given:

- `concurrentRequests: "100" && maxReplicas: 10` with 200rps should scale to approx. 2
- `concurrentRequests: "10" && maxReplicas: 10`  with 10rps should scale to approx. 1

The reason I say approximately is that the aggregation of the number of requests per seconds happens every 15 seconds, and depending on the actual/average rate a client is sending requests at, things might be a bit different. 


here is a screen capture for an app with the following scale configuration

```json
{
  "maxReplicas": 10,
  "minReplicas": 1,
  "rules": [
    {
      "http": {
        "metadata": {
          "concurrentRequests": "2"
        }
      },
      "name": "http"
    }
  ]
}
```

> hey `-c` is number of clients, `-q` is queries (or requests) per second per client, `-z` is time

You'll see that:

- 1 client, 1 rps  ---> 1 replica
- 1 client, 3 rps  ---> 2 replicas
- 1 client, 10 rps ---> 5 replicas
- 4 clients, 5 qpc (4*5 = 20 rps) --> 10 replicas

_you'll notice that I tried the last example with -c 1 -q 20, but hey was only able to send ~14 rps and I needed to fiddle with it a bit_

![output](https://user-images.githubusercontent.com/645740/221015539-3dad74b1-3e7d-4fea-894e-b881dfe213a4.gif)
